### PR TITLE
Second most repeated character in a sequence

### PR DESCRIPTION
**Pull request by SKAUL05**
This `PR` is about adding an algorithm which would find the second most repeated character in a sequence

### CODE REVIEW

**Comment by SKAUL05** [comment]
> How about generalizing the algorithm to find **kth** most repeated character in a sequence

@onlinejudge95 Changed to support nth repeated

File: other/nth_repeated.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Created by sarathkaul on 04/02/20
+# Script to print second most repeated character in sequence
+from collections import Counter
+
+
+def second_repeated(sequence: list, nth_repeated: int) -> str:
```
Comment (onlinejudge95) [inline other/nth_repeated.py]: Change the method name`

File: other/nth_repeated.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    """
+    >>> sequence = ['Algorithms','Algorithm','Python','Python','The','Python','The']
+    >>> nth_repeated = 2
```
Comment (onlinejudge95) [inline other/nth_repeated.py]: What if `nth_repeated` is < 1.

**Comment by SKAUL05** [comment]
@onlinejudge95 Changes done

**Comment by cclauss** [comment]
The algorithm is a one liner...
```python
def count_repeated(sequence: list, nth_repeated: int) -> str:
        return "" if nth_repeated < 1 else Counter(sequence).most_common()[nth_repeated - 1][0]
```

**Comment by cclauss** [comment]
Sorry @ SKAUL05 but closing as this has now become a demonstration of how to use the Python Standard Library rather than an algorithm.  Please do not be discouraged because this has been a learning experience.  I hope you will make other submissions of algorithms in the near future.

**Pull request closed without merging**